package.json

```diff
@@ -89,6 +91,7 @@
     "add:devdepbysha": "bash -c 'yarn add -W --dev \"git+https://git@github.com/Khan/$0.git#$1\"'"
   },
   "dependencies": {
-    "@rollup/plugin-terser": "^0.4.0"
+    "@rollup/plugin-terser": "^0.4.0",
+    "@types/superagent": "^4.1.16"
```

Comment:
I've been treating `@types` as dev dependencies.  This is what is shown on https://github.com/DefinitelyTyped/DefinitelyTyped#npm.

Reply:
Hmm, this is likely just a typo. I can fix that.